### ISSUE TITLE
Windows High Contrast Mode support

### ISSUE THREAD

**Issue by jnurthen** [adobe/spectrum-web-components#297]
### Expected Behaviour
Windows High contrast mode should be usable.
Note: It will not be pretty but we should be able to 
* See focus
* See any "selection" 

### Actual Behaviour
Focus is not visible on some components

### Reproduce Scenario (including but not limited to)
* Use IE or Firefox on windows (Chrome does not support Windows HCM - MSFT is working on this support as part of Chrome uptake)
* Turn on HCM (left shift+ left alt+ Print Screen)
* Run the demo for tabs

#### Steps to Reproduce
Note that there is no visible focus when moving to the tab.

#### Platform and Version
Win
FF, IE or Edge (non Chrome)

#### Sample Code that illustrates the problem
Tabs demo in storybook

#### Additional Notes
Windows HCM will always render borders. These components have a transparent border when not focused and a visible border when focused. In HCM this is the same.
Fix by removing the transparent border when the item does not have focus.

**Comment by Westbrook** [comment]
Investigate transparent border usage to support this as a quick workaround.

**Comment by Westbrook** [comment]
This last bit of coverage with the tabs pattern looks addressed in https://github.com/adobe/spectrum-css/pull/1176, hopefully we can get a Spectrum CSS release with those changes consumes here, soon!

**Comment by Westbrook** [comment]
There's definitely more work to be done here as a whole, however, as we're now actively acquiring this styling from Spectrum CSS without interference from our team as it becomes available, I'm going to close this issue.
